packages/esm-patient-chart-app/src/workspace/workspace-window.scss

```diff
@@ -115,7 +115,7 @@
 
   .fixed {
     top: 3rem;
-    bottom: spacing.$spacing-10;
+    bottom: calc(spacing.$spacing-10 + 1px); // plus the top border
```

Comment:
Does using the below property avoid the +1px?
Please try and let me know
Thanks!
```
box-sizing: border-box;
```

Reply:
tried this, but the top border is not shown when a workspace is opened